website/blog/2024-02-29-StateFlow/index.mdx

```diff
@@ -135,7 +135,7 @@ When returning `None`, the group chat will terminate.
 
 
 ## For Further Reading
-* [StateFlow notebook](https://microsoft.github.io/autogen/docs/notebooks/agentchat_groupchat_stateflow/)
-* [GroupChat with Customized Speaker Selection notebook](https://microsoft.github.io/autogen/docs/notebooks/agentchat_groupchat_customized/)
+* [StateFlow notebook](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_groupchat_stateflow.ipynb)
+* [GroupChat with Customized Speaker Selection notebook](https://github.com/microsoft/autogen/blob/main/notebook/agentchat_groupchat_customized.ipynb)
```

Comment:
Isn't it better to use the web-rendered notebook?
Also, we can use paths like (/docs/...)